src/labels_transfer/api/common_arguments.yaml

```diff
@@ -25,7 +24,7 @@
                     obsm:
                       - type: "double"
                         name: "features"
-                        example: X_integrated_scanvi
+                        example: X_scvi
```

Comment:
Just a question for my own understanding: why was the previous example worse?

Reply:
That was Dorien's suggestion, so I just kept it. I guess it is more common